src/main/java/org/primeframework/mvc/action/config/ActionConfiguration.java

```diff
@@ -60,6 +60,8 @@ public class ActionConfiguration {
 
   public final Map<HTTPMethod, ExecuteMethodConfiguration> executeMethods;
 
+  public final String fallbackMessageResourcesPath;
```

Comment:
probably should be `fallbackActionMessageUri` ?

Reply:
If we do use that name, it would be `fallbackActionMessageURI` (the `URI` should be ucase)

Reply:
Fixed.